profiles/package.mask

```diff
@@ -33,6 +33,10 @@
 
 #--- END OF EXAMPLES ---
 
+# Volkmar W. Pogatzki <gentoo@pogatzki.net> (2022-04-22)
+# Unused java library. Removal in 30 days.
```

Comment:
I really suggest to always specific an absolute removal date, as it makes the treecleaners jobs much easier.